nvtabular/workflow/workflow.py

```diff
@@ -248,6 +245,8 @@ def fit(self, dataset: Dataset) -> "Workflow":
         output_dtypes = self.transform(dataset).sample_dtypes()
         self.output_dtypes = dict(zip(output_dtypes.index, output_dtypes))
 
+        self._zero_output_schemas()
+        self.fit_schema(dataset.infer_schema())
```

Comment:
Looks like this requires fitting the schema twice? That's probably fine for now, but I wonder if we should split `fit_schema` into computing columns/tags vs computing properties at some point in the future.